bin/demeuk.py

```diff
@@ -135,6 +137,8 @@
 HASH_PHPBB_REGEX = '^\\$[hH]\\$[\\w\\.\\/]{6,}$'
 HASH_REGEX_LIST = [HASH_BCRYPT_REGEX, HASH_CRYPT_SALT_REGEX, HASH_CRYPT_REGEX, HASH_PHPBB_REGEX]
 
+TRIM_BLOCKS = ('\\\\n', '\\\\r', '\\n', '\\r', '<br>', '<br />')
```

Comment:
Small note on the `<br />`
`<br/>` is also valid for example. Are they present in your datasets?
Same goes for `<br></br>`. Are they present in your datasets?

Reply:
Discussed with @rickvanderzwet, in his dataset he did not run into the other `br` blocks, so I have merged the code as is.